### ISSUE TITLE
Add reverb effects

### ISSUE THREAD

**Issue by rishavk1102** [lincollincol/AudioTool#4]
Hello, I am using your library and it is literally amazing. My workload has reduced 10x times.

When I am trying to apply reverb at any value, the output is very faint and the log is as follows: 

> Lavc58.65.102 libmp3lame
> 2021-06-08 17:35:41.128 23842-13241/com.mehoot.audiotesting I/mobile-ffmpeg: size=      41kB time=00:00:02.59 bitrate= 130.3kbits/s speed=31.9x    
> 2021-06-08 17:35:41.128 23842-13241/com.mehoot.audiotesting I/mobile-ffmpeg: video:0kB audio:41kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 
> 2021-06-08 17:35:41.128 23842-13241/com.mehoot.audiotesting I/mobile-ffmpeg: 0.869648%

What is the recommended way of applying reverb to using this library?

Please help!

**Comment by MirzaQais** [comment]
@rishavk1102  had you applied the reverb effect using this library ?

**Comment by rishavk1102** [comment]
No, I shifted to the FFMPEG library, which is much more flexible.

**Comment by MirzaQais** [comment]
Brother I want to apply
Reverb
Auto tune
Distortion and echo effects on audio file
Can you please help me with this?
I am stuck in this from last two weeks

**Comment by MirzaQais** [comment]
ok i want to apply the reverb effect with parameters large hall small and huge
can you help me with this?
